edx_event_bus_kafka/internal/producer.py

```diff
@@ -328,14 +329,9 @@ def poll_indefinitely(api_weakref: KafkaEventProducer):
             api_object = None
 
 
-# Note: This caching is required, since otherwise the Producer will
-# fall out of scope and be garbage-collected, destroying the
-# outbound-message queue and threads. The use of this cache allows the
-# producer to be long-lived.
-@lru_cache  # will just be one cache entry, in practice
 def get_producer() -> Optional[KafkaEventProducer]:
```

Comment:
I think I should change this to `create_producer` to reflect that it's not cached.